calour/io.py

```diff
@@ -224,7 +224,11 @@ def _read_metadata(ids, f, kwargs):
         else:
             kwargs['dtype'] = {index_col: str}
 
-        metadata = pd.read_table(f, **kwargs)
+        try:
+            metadata = pd.read_table(f, **kwargs)
```

Comment:
why this instead of letting it fail?

Reply:
I saw several users encountering a problem with reading the metadata, and the regular error message is confusing for non-advanced python users (who don't know the calour code). This just gives them a general message the mapping file has a problem and then raises the regular python error, so less confusing for them (the error message is the first line they see when it fails...).